lib/libutee/include/k3/otp_keywriting_ta.h

```diff
@@ -0,0 +1,51 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Texas Instruments System Control Interface Driver
+ *
+ * Copyright (C) 2023 Texas Instruments Incorporated - https://www.ti.com/
+ *	Manorit Chawdhry <m-chawdhry@ti.com>
+ */
+
+#ifndef __OTP_KEYWRITING_H__
+#define __OTP_KEYWRITING_H__
+
+/* UUID of the trusted application */
+#define PTA_K3_OTP_KEYWRITING_UUID \
+	{ 0xacc50f40, 0x0613, 0x4abd, \
+		{ 0x8d, 0xfe, 0xa9, 0x64, 0xcb, 0x74, 0xeb, 0x69} }
+
+#define PTA_K3_OTP_KEYWRITING_NAME "pta_k3_otp.ta"
+
+/*
+ * TA_OTP_KEYWRITING_CMD_READ_RAW - Read an extended OTP bit
```

Comment:
s/TA_OTP_KEYWRITING_CMD_READ_RAW/TA_OTP_KEYWRITING_CMD_READ_MMR/

Same comment for `TA_OTP_KEYWRITING_CMD_WRITE_ROW` description below.